### DISCUSSION TITLE
School map not showing

### DISCUSSION THREAD

**Discussion by aber007** [community/community#39335] [GitHub Education]
Hello. I just moved and therefore switched school email address. When I was about to add my school it doesn't show up even though schools from the same "company" does. Anyways, when I pressed continue it asked me to put a marker on a map but this map doesn't show up at all. Is this a bug or am I just dumb?
I also accidentally pressed another school within the same community by accident (AISM instead of AISL). Is it possible to switch school of an email address?
![Screenshot 2022-11-17 162108](https://user-images.githubusercontent.com/60569011/202486211-f54d6ce1-599d-40eb-aa3f-092db260f9f9.png)

**Comment by aber007** [comment]
So the first part is solved but I still have problem with the second one.

**Reply by ghost** [reply]
How did you solve the first problem!

**Comment by samus-aran** [comment]
Hi @aber007 👋 

How did you get on? Are you still having issues with selecting the correct school?

**Reply by ghost** [reply]
I need help with the same issue! I want to link this account to my school email, and halfway through the application process, the school map isn't appearing!

**Comment by ghost** [comment]
Have you tried typing in the full address? It might work...
